.travis.yml

```diff
@@ -84,6 +84,12 @@ jobs:
       - bin/check_swagger
       name: "swagger check"
 
+    - script:
+      - bin/build_xgo linux/arm
+      - bin/builder_run BINARY=build/myst/myst_linux_arm bin/package_debian $BUILD_VERSION armhf
```

Comment:
Are we adding ARM .deb to release artifacts also?

Reply:
yes, but this PR is super draft just for CI experiments. It's not ready for review. :D